python/tskit/drawing.py

```diff
@@ -460,11 +461,21 @@ def __init__(
         self.edge_attrs = {}
         self.node_attrs = {}
         self.node_label_attrs = {}
+        symbol_size = "{:g}".format(rnd(self.symbol_size))
+        half_symbol_size = "{:g}".format(rnd(self.symbol_size / 2))
         for u in tree.nodes():
             self.edge_attrs[u] = {}
             if edge_attrs is not None and u in edge_attrs:
                 self.edge_attrs[u].update(edge_attrs[u])
-            self.node_attrs[u] = {"r": "{:g}".format(rnd(self.symbol_size / 2))}
+            self.node_attrs[u] = {
```

Comment:
This is a bit obscure - wouldn't it be easier to have a ``sample_node_attrs`` variable or something? Having to demultiplex the information and filter the non-relevant bits out is pretty messy and error prone (i.e, what happens when someone adds an extra attr to the circle? )

Reply:
Yes, it's a bit fiddly. But I assume that most people will *want* the same thing applied to both sample nodes and other nodes (e.g. `"stroke":"black", "fill":"white"`). I think all the other attributes are identical across rects and circles. Would we then have an extra param in the API (`sample_node_attrs=None`), and even a `sample_node_label_attrs=None` to match? Seems like it's growing a bit out of control.

Or do you mean we do this internally, but don't expose it to the user?

Reply:
I mean internally, so the code is clear.